streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1568,21 +1105,19 @@ private Topology createSimpleTopologyWithDroppingPartitioner() {
                 .addSink("sink", OUTPUT_TOPIC_1, new DroppingPartitioner(), "processor");
     }
 
-    @Deprecated // testing old PAPI
     private Topology createStatefulTopology(final String storeName) {
         return topology
             .addSource("source", STRING_DESERIALIZER, STRING_DESERIALIZER, INPUT_TOPIC_1)
-            .addProcessor("processor", define(new OldAPIStatefulProcessor(storeName)), "source")
+            .addProcessor("processor", () -> new StatefulProcessor(storeName), "source")
```

Comment:
Indeed, it looks like `OldAPIStatefulProcessor` has been corrected to `StatefulProcessor` for migrating to current API. Got it! Thanks for teaching!